doc/playground.lua

```diff
@@ -0,0 +1,145 @@
+local console = require('console')
+local vshard = require('vshard')
+local crud = require('crud')
+
+-- Trick to don't leave *.snap, *.xlog files. See
+-- test/tuple_keydef.test.lua in the tuple-keydef module.
+if os.getenv('KEEP_DATA') ~= nil then
+    box.cfg()
+else
+    local tempdir = fio.tempdir()
+    box.cfg({
+        memtx_dir = tempdir,
+        wal_mode = 'none',
+    })
+    fio.rmtree(tempdir)
+end
+
+-- Setup vshard.
+_G.vshard = vshard
+box.once('guest', function()
+    box.schema.user.grant('guest', 'super')
+end)
+local uri = 'guest@localhost:3301'
+local cfg = {
+    bucket_count = 3000,
+    sharding = {
+        [box.info().cluster.uuid] = {
+            replicas = {
+                [box.info().uuid] = {
+                    uri = uri,
+                    name = 'storage',
+                    master = true,
+                },
+            },
+        },
+    },
+}
+vshard.storage.cfg(cfg, box.info().uuid)
+vshard.router.cfg(cfg)
+vshard.router.bootstrap()
+
+-- Create the 'customers' space.
+box.once('customers', function()
+    box.schema.create_space('customers', {
+        format = {
+            {name = 'id', type = 'unsigned'},
+            {name = 'bucket_id', type = 'unsigned'},
+            {name = 'name', type = 'string'},
+            {name = 'age', type = 'number'},
+        }
+    })
+    box.space.customers:create_index('primary_index', {
+        parts = {
+            {field = 1, type = 'unsigned'},
+        },
+    })
+    box.space.customers:create_index('bucket_id', {
+        parts = {
+            {field = 2, type = 'unsigned'},
+        },
+    })
+    box.space.customers:create_index('age', {
+        parts = {
+            {field = 4, type = 'number'},
+        },
+    })
+
+    -- Fill the space.
+    box.space.customers:insert({1, 477, 'Elizabeth', 12})
+    box.space.customers:insert({2, 401, 'Mary', 46})
+    box.space.customers:insert({3, 2804, 'David', 33})
+    box.space.customers:insert({4, 1161, 'William', 81})
+    box.space.customers:insert({5, 1172, 'Jack', 35})
+    box.space.customers:insert({6, 1064, 'William', 25})
+    box.space.customers:insert({7, 693, 'Elizabeth', 18})
+end)
+
+-- Create the developers space.
+box.once('developers', function()
+    box.schema.create_space('developers', {
+        format = {
+            {name = 'id', type = 'unsigned'},
+            {name = 'bucket_id', type = 'unsigned'},
+            {name = 'name', type = 'string'},
+            {name = 'surname', type = 'string'},
+            {name = 'age', type = 'number'},
+        }
+    })
+    box.space.developers:create_index('primary_index', {
+        parts = {
+            {field = 1, type = 'unsigned'},
+        },
+    })
+    box.space.developers:create_index('bucket_id', {
+        parts = {
+            {field = 2, type = 'unsigned'},
+        },
+    })
+    box.space.developers:create_index('age_index', {
+        parts = {
+            {field = 5, type = 'number'},
+        },
```

Comment:
And another one non-unique index here:
```suggestion
        parts = {
            {field = 5, type = 'number'},
            unique = false,
        },
```

Reply:
Fixed (see the comment above).